src/pages/kb/user-guide/alerts/setting-up-an-alert.md

```diff
@@ -14,43 +15,112 @@ title: Setting Up An Alert
+
+Finally add an [alert destination]({% link _kb/user-guide/alerts/creating-new-alert-destination.md %}) and save your alert. If you skip this step you will not receive any notifications when the alert is triggered.
+
+![](/assets/images/docs/gitbook/alert_destination.png)
+
+# Alert Status & Frequency
+
+The status of your Alert criteria is checked every time the query is executed. Alerts have three possible statuses:
+
+* `TRIGGERED` means that your query matched the criteria defined in your alert configuration. If you set your alert to trigger when the value of "cats" is greater than 1500 as long as it's above 1500 your alert is triggered.
+* `OK` means that the most recent query execution did not match the criteria defined in your alert. This doesn't mean that the Alert has not been triggered previously. If your "cats" value is now 1470 your alert will show as OK.
+* `UNKNOWN` means the query has not executed since you configured the Alert. You should see this status immediately after creating your Alert. The status will always show triggered or OK after that. If it does not change following your next query execution then review the above instructions as you have likely configured the Alert incorrectly.
+
+Redash sends notifications to your chosen Alert Destinations whenever it detects that the Alert status has changed from `OK` to `TRIGGERED` or vice versa. Consider this example where an Alert is configured on a query that is scheduled to run once daily. The daily status of the Alert appears in the table below. Prior to Monday the alert status was `OK`.
+
+| Day       | Alert Status | 
+|-----------|--------------| 
+| Monday    | OK           | 
+| Tuesday   | OK           | 
+| Wednesday | TRIGGERED    | 
+| Thursday  | TRIGGERED    | 
+| Friday    | TRIGGERED    | 
+| Saturday  | TRIGGERED    | 
+| Sunday    | OK           | 
+
+By default, Redash would send a notification on Wednesday when the status changed from `OK` to `TRIGGERED` and again on Sunday when it switches back. It will not send alerts on Thursday, Friday, or Saturday unless you specifically configure it to do so because the Alert status did not change between executions on those days.
+
+## Setting Rearm Seconds for an Alert
+
+To send notifications more frequently, set the **Rearm seconds** to any nonzero value. This tells Redash to treat the next query execution after the chosen number of seconds as though the status has changed and therefore send a notification.
+
+For example, if the Alert above were configured with **Rearm seconds** equal to `1` a notification would be sent every day. Normally, Redash would see the `OK` status on Monday but would not send a notification because the value the previous day was also `OK`. But with _Rearm seconds_ elected, Redash will treat _any status_ as though it were different from the previous execution and send a notification anyway.
```

Comment:
"Rearm seconds" is probably the most confusing and hard to explain feature we ever had.

In the scenario above, you will get a notification on Wednesday, Thursday, Friday, Saturday and last one on Sunday. You will not receive on Monday, because we only resend a notification when it's triggered...